### PR TITLE
 fix(curriculum): updated description, tests and styles for calorie counter

### LINKED ISSUE #53033: Incorrect Use of Terms in Calorie Counter Project
### Describe the Issue

We got an email in the support inbox recently about the incorrect use of the terms "surplus" and "deficit" in the calorie counter project.

According to [this source](https://www.pcyc.org.au/blog/nutrition-tip-what-is-a-caloric-surplus/), a caloric surplus is when you consume more calories than you expend throughout the day, and a deficit is when you consume less than you expend.

Currently the calorie counter project uses these terms the opposite way. It shows "x Calorie Surplus" when the number of calories remaining is less than the daily calorie budget, and "x Calorie Deficit" the number of calories consumed goes over the daily budget.

I think we have a couple of options here:

1. Update the description and seeds starting from step 77 (currently id 63c9e55b4b06c11fff555c64) to the final step so the ternary operator `surplusOrDeficit` returns `Surplus` if `remainingCalories` is less than `0`, and returns `Deficit` if not (`const surplusOrDeficit = remainingCalories < 0 ? 'Surplus' : 'Deficit';`).
2. Since the terms `Surplus` and `Deficit` can still be confusing, update the project to use the text "X Calories Remaining" / "X Calories Over" or something similar, and for the text color to be green if `remainingCalories` is greater than or equal to `0`, and red if not. The variable name `surplusOrDeficit` could still be used, but it may be better to update it to something more general like `overOrUnder`. If we do update this variable, we should also update the classes in the HTML and CSS from `surplus` and `deficit` to `over` and `under`, too.

It would sound nicer to use "X Calories Remaining" / "-X Calories Remaining", but I believe this project is the only one to teach the `Math.abs()` method at the moment, so wording like "X Calories Remaining" / "X Calories Over" would allow us to keep teaching it here.

Or if someone has another suggestion for a way to improve this project, feel free to share your thoughts and we can discuss it.

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-form-validation-by-building-a-calorie-counter/step-77

Edit: Updated the text and ternary operator so `Surplus` is returned when `remainingCalories` is less than `0` rather than less than or equal to `0`. There doesn't seem to be a term for when the amount of calories consumed is the same as calories burned / when remaining calories is 0, but a "0 calorie deficit" seems closer since you should maintain your current weight.

~~`const surplusOrDeficit = remainingCalories <= 0 ? 'Surplus' : 'Deficit';`~~ --> `const surplusOrDeficit = remainingCalories < 0 ? 'Surplus' : 'Deficit';`

### PR DESCRIPTION
**Pull request by RGHANILOO**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #53033

### CODE REVIEW

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/5ddb965c65d27e1512d44d9a.md
Comment (RGHANILOO) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/5ddb965c65d27e1512d44d9a.md:150]: Suggested styling

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63b606f09a14cc1781aea1fb.md
Comment (RGHANILOO) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63b606f09a14cc1781aea1fb.md:174]: Suggested styling

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e55b4b06c11fff555c64.md
Comment (RGHANILOO) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e55b4b06c11fff555c64.md:25]: tests were updated to reflect the new ternary operator

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e55b4b06c11fff555c64.md
Comment (RGHANILOO) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e55b4b06c11fff555c64.md:22]: updated the testes and the helpers based on the new information

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e5eea8261d22856ead1c.md
Comment (RGHANILOO) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e5eea8261d22856ead1c.md:258]: updated ternary operator manually to ensure linting did not override.

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9f2bff625af342023512c.md
Comment (RGHANILOO) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9f2bff625af342023512c.md:540]: both seed and solution updated with correct operator and indents

**Review by RGHANILOO — COMMENTED** [review]
i have individually reviewed any of the changes i have manually made to ensure there are no indentation issues this time round

**Review by scissorsneedfoodtoo — APPROVED** [review]
Thanks again for all of your help with this, @RGHANILOO.

Tested locally and everything LGTM 👍

**Comment by jdwilkin4** [comment]
Hey @RGHANILOO ! 

It looks like Kris approved your changes but then you re requested another review.
It doesn't look like you made any additional changes since then.

Was there a reason for the re review?

If so, you should reply back so Kris can narrow down the scope of where to look

**Comment by RGHANILOO** [comment]
> Hey @RGHANILOO !
> 
> It looks like Kris approved your changes but then you re requested another review. It doesn't look like you made any additional changes since then.
> 
> Was there a reason for the re review?
> 
> If so, you should reply back so Kris can narrow down the scope of where to look

No all is good , I think I may have accidentally requested an approval.
Didn't even realise you could request one as I assumed that would be a rude thing to do. But I think it's still waiting for a second approval .
Thanks for highlighting it .

**Review by jdwilkin4 — APPROVED** [review]

**Pull request merged**
